### ISSUE TITLE
Skype keeps refreshing constantly

### ISSUE THREAD

**Issue by aaronfranke** [stanfieldr/ghetto-skype#25]
Here's a video: https://www.youtube.com/watch?v=yGb-6Z_BQw8

The refreshing never stops, it just keeps refreshing. I let it sit for 5 minutes just to be sure.

This occurs on Ghetto Skype 1.3.0 and 1.2.3. This issue does not occur on version 1.2.2.

I have tried deleting the folder `~/.config/Ghetto Skype` and using `sudo apt purge -y ghetto-skype` each time to be sure I'm starting with a clean slate when I re-install it but this issue still occurs.

**Comment by stanfieldr** [comment]
Did it work before or after you tried 1.3.0? I just noticed Skype just changed it's login screen. Also, what does your settings file look like? Are you using a proxy?

Thanks,
Robert

**Comment by aaronfranke** [comment]
The reason I tried to update Ghetto Skype is because I seem to be unable to make any calls on 1.2.2 and so I'd like to be able to update but the newer versions don't seem to be working because of this issue.

**Comment by ilusi0n** [comment]
I can confirm this bug. The interface always keep refreshing which makes impossible to login.

**Comment by whitephoenix0** [comment]
Confirmed on fully updated arch linux.

**Comment by stanfieldr** [comment]
Would it be possible for one of yall to do the following for me:

```bash
$ git clone -b connection-test https://github.com/stanfieldr/ghetto-skype.git
$ cd ghetto-skype
$ npm install && npm start > output.txt
```

I made a minor change that may have fixed the issue, but if not, please upload the output.txt file.

**Comment by ilusi0n** [comment]
I just tested it. It works now. Thanks!

**Comment by stanfieldr** [comment]
Awesome glad to hear. I added the fix to [v1.3.1](https://github.com/stanfieldr/ghetto-skype/releases/tag/v1.3.1)

**Comment by aaronfranke** [comment]
I can confirm that this bug is now fixed.
